plugins/inara.py

```diff
@@ -918,40 +1179,57 @@ def worker():
                                 plug.show_error(_('Error: Inara {MSG}').format(
                                     MSG=f'{data_event["eventName"]},'
                                         f'{reply_event.get("eventStatusText", reply_event["eventStatus"])}'))
+
                         if data_event['eventName'] in ('addCommanderTravelCarrierJump',
                                                        'addCommanderTravelDock',
                                                        'addCommanderTravelFSDJump',
                                                        'setCommanderTravelLocation'):
                             this.lastlocation = reply_event.get('eventData', {})
                             # calls update_location in main thread
                             this.system_link.event_generate('<<InaraLocation>>', when="tail")
+
                         elif data_event['eventName'] in ['addCommanderShip', 'setCommanderShip']:
                             this.lastship = reply_event.get('eventData', {})
                             # calls update_ship in main thread
                             this.system_link.event_generate('<<InaraShip>>', when="tail")
 
                 break
+
             except Exception as e:
                 logger.debug('Sending events', exc_info=e)
                 retrying += 1
         else:
             if callback:
                 callback(None)
+
             else:
                 plug.show_error(_("Error: Can't connect to Inara"))
 
 
-# Call inara_notify_location() in this and other interested plugins with Inara's response when changing system or station
+# TODO(A_D) event is unused
```

Comment:
It's a Tk event I believe, so having the parameter is mandatory.

Reply:
if its mandatory it can be replaced with _, so not to imply its useful

Reply:
either way Ill remove the comment